stdlib/toolchain/legacy_layouts/CMakeLists.txt

```diff
@@ -55,11 +55,10 @@ foreach(sdk ${SWIFT_SDKS})
   endforeach()
 endforeach()
 
-# Bootstrapping - level 0
-
-add_layout_copying( ${SWIFT_HOST_VARIANT_SDK} ${SWIFT_HOST_VARIANT_ARCH} "0")
-
-# Bootstrapping - level 1
-
-add_layout_copying( ${SWIFT_HOST_VARIANT_SDK} ${SWIFT_HOST_VARIANT_ARCH} "1")
+if(${BOOTSTRAPPING_MODE} STREQUAL "BOOTSTRAPPING.*")
```

Comment:
I believe `STREQUAL` will not work here. It needs to be `MATCHES`.
Did you test this change?